### PR TITLE
Docs

### LINKED ISSUE #35: Update Docs
### Description
Docs would be nice

### What I Did

Docs added to `usage.rst` file demonstrating interactive and batch mode use of `jrnr`

### PR DESCRIPTION
**Pull request by jgerardsimcock**
- [x] closes #35 
 - [ ] tests added / passed
 - [x] docs reflect changes
 - [x] passes ``flake8 jrnr tests docs``
 - [x] entry in HISTORY.rst

Adds documentation to use jrnr in interactive mode as well as batch mode.

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -2,4 +2,144 @@
...
+        write_variable:  tas
+        description:     Average Daily Temperature, tavg\n\n Average Daily Temper...
+        execute:         python tas.py run
+        project:         gcp
+        team:            climate
+        dependencies:    ['/global/scratch/groups/co_laika/gcp/climate/nasa_bcsd/...
+        model:           NorESM1-M
+
+
+As you can see, if you setting up logging, the logging information will print to the terminal to your terminal se
```
Comment (delgadom) [inline docs/usage.rst]: to your terminal se...?

File: docs/tas.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
+'''
```
Comment (delgadom) [inline docs/tas.py:1]: should this file be here?

File: docs/usage.rst [hunk 3]
```diff
@@ -2,4 +2,144 @@
...
+
+Notice that we use the unique id `001` and the jobname `tas` that we used when we created the job. You must use these values or we cannot compute the progress of our job.
+
+
+Technical note
+--------------
+
+How does this `jrnr` track the status of my jobs? 
+
+In your directory where you are running your job, `jrnr` creates a `locks` directory. In this `locks` directory, for each job in your set of batch jobs a file is created with the following structure `{job_name}-{unique_id}-{job_index}.` When a node is working on a job, it adds the `.lck` file extension to the file. When the job is completed, it converts the `.lck` extension to a `.done` extension. If, for some reason, the job encounters an error, the extension will shift to `.err`. When you call the `status` command `jrnr` is just displaying the count of files with each file extension in the locks directory.
```
Comment (delgadom) [inline docs/usage.rst]: this whole file is fantastic. thank you!!!

**Review by delgadom — APPROVED** [review]

File: docs/tas.py (hunk 2 above)
Comment (jgerardsimcock) [reply docs/tas.py:1]: This file I want show as a template for setting up a jrnr script. We can also link to impactlab documentation on gitlab for more examples but I wanted to have something in the docs.

File: docs/usage.rst (hunk 1 above)
Comment (jgerardsimcock) [reply docs/usage.rst]: nice catch!

File: docs/tas.py (hunk 2 above)
Comment (delgadom) [reply docs/tas.py:1]: ok. then can you link to this file somewhere in the actual docs? right now I don't see it anywhere in the navigation.

**Review by delgadom — CHANGES_REQUESTED** [review]

File: docs/tas.py (hunk 2 above)
Comment (jgerardsimcock) [reply docs/tas.py:1]: Just pushed. There is now a tas.rst file that references the tas.py file. I followed the setup from this: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google

Comment (delgadom) [reply docs/tas.py:1]: but there is no way to access this file. have you tried building it and using the site? as far as I can tell, the user can't get to tas.rst.

You could either include tas.rst in the table of contents, or you could include a link to it in the usage file. Or both. But right now this file is truly an orphan. The other way you could do this is to allow the user to download tas.py, with a download directive. I'm agnostic about how you do it, but right now this page is inaccessible. There are a bunch of examples of including python scripts in a documentation page in the documentation repo, as well as the datafs docs. Let me know if you want help with this, but it should just be a matter of doing an internal hyperlink to the tas.rst header.

**Review by delgadom — CHANGES_REQUESTED** [review]

**Comment by delgadom** [comment]
looks like you removed `modules.rst` from the index, but didn't remove the file? and why did you remove modules.rst? that's how the autodoc works

on travis:
/home/travis/build/ClimateImpactLab/jrnr/docs/modules.rst:document isn't included in any toctree

**Comment by delgadom** [comment]
probably running `sphinx-apidoc` again would be a good idea, just to make sure all the modules/methods get included in the documentation

**Review by delgadom — APPROVED** [review]

**Pull request merged**